src/hooks/useAvailableRepositories.js

```diff
@@ -1,25 +1,41 @@
 import { useQuery } from 'react-query';
 import useChrome from '@redhat-cloud-services/frontend-components/useChrome';
 
-const fetchAdditionalRepositories = (token) => async (keyName) => {
+const fetchAdditionalRepositories = async (
+  keyName,
+  offset,
+  allRepositories = []
+) => {
   if (!keyName) {
     return false;
   }
 
+  const token = await window.insights.chrome.auth.getToken();
+
```

Comment:
This is undoing the work just done to move away from `window.insights.chrome.auth.getToken()` completed in #191